docs/docs/03-tools/05-context.md

```diff
@@ -0,0 +1,64 @@
+# Context
+
+GPTScript supports context provider tools which can be used to provide additional context for tool execution. Output from the context tool gets prepended to the instruction of the current tool that uses the context tool.
+
+## Writing a Context Provider Tool
+
+Here is a simple example of a context provider tool that provides additional context to search tool:
+
+```yaml
+# my-search-context-tool.gpt
+name: search_context
+export: sys.http.html2text?
+
+#!/bin/bash
+echo You are an expert web researcher with access to the Search tool.If the search tool fails to return any information stop execution of the script with message "Sorry! Search did not return any results". Feel free to get the contents of the returned URLs in order to get more information. Provide as much detail as you can. Also return the source of the search results.
+
+```
+
+## Using a Context Provider Tool
+
+Continuing with the above example, this is how you can use it in a script:
+
+```yaml
+context: search_context from  my-search-context-tool.gpt
```

Comment:
change to `context: ./my-search-context-tool.gpt`

Reply:
Made the suggested change.